lib/resources/example_json/r4_examples_communication.rb

```diff
@@ -94,5 +94,50 @@ module Resources
       ]
     }
 
+    R4_COMMUNICATION_CREATE ||= {
+      "resourceType": "Communication",
+      "extension": [
+        {
+            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/reply-to",
+            "valueReference": {
+                "reference": "Group/6345478364"
+            }
+        }
+      ],
+      "status": "completed",
+      "category": [{ 	
```

Comment:
2 patterns going on right now. 1 where the brackets are all spread out, the other where they are grouped. Pick one or the other.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/435/commits/720c35bf50e9cb8dd953ad8a2bce0b6a3c29f74a#diff-ebb0057807d83ddfb753281899d4d3b5L108-R146